src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = [];
```

Comment:
Can you maybe name a variable in the other way? In your function you return an array with history of states after actions. That`s why it is better to name a variable like "stateHistory" or "historyOfStates".